docs/api-reference/self-signed-certificates.md

```diff
@@ -0,0 +1,79 @@
+<!--pytest-codeblocks:skipfile-->
+<!-- markdownlint-disable-next-line first-line-h1 -->
+When using the `quilt3` API or CLI with a client-to-site (remote
+access) VPN you may need to use a self-signed certificate to
+securely connect. This requires adding the certificate to your
+Python environment.
+
+## Mac OS X
+
+1. Open a new browser window and navigate to any HTTPS website.
+2. Click the «lock icon» in the browser navigation/address bar.
+3. Click «View certificates» and record the certificate name value
+(e.g. `mycert`).
+4. Open Keychain Access.
+5. Select System Keychains > System Roots.
+6. Select the «Certificates» tab.
+7. Search all certificates for a certificate name value that equals
+the value recorded (`mycert` in the example).
+8. Click «File», select "Export Items...", export the root certificate
+to your localhost.
+9. In the Terminal, convert the exported certificate with
+[OpenSSL](https://www.openssl.org/):
+    ```sh
+    openssl x509 -inform der -in /path/to/your/certificate.cer -out /path/to/converted/certificate.crt
+    ```
+10. In the Terminal, open your shell profile (e.g. `~/.bashrc` or
+`~/.zshrc`).
+11. Add the certificate to your shell environment:
+    ```ssh
+    export REQUESTS_CA_BUNDLE=/path/to/converted/certificate.crt`
+    ```
```

Comment:
```suggestion
10. Export the following variable (e.g. in `~/.zshrc`):
    ```sh
    export REQUESTS_CA_BUNDLE=/path/to/converted/certificate.crt`
    ```
```

Reply:
Fixed the stray `. Leaving bashrc and zshrc in place (but in simplified format per Aneesh)